acestep/api_server.py

```
@@ -213,527 +216,122 @@ def _path_to_audio_url(path: str) -> str:
+        runtime = initialize_lifespan_runtime(
+            app=app,
+            store=store,
+            queue_maxsize=QUEUE_MAXSIZE,
+            avg_window=AVG_WINDOW,
+            initial_avg_job_seconds=INITIAL_AVG_JOB_SECONDS,
+            get_project_root=_get_project_root,
+            initialize_training_state_fn=initialize_training_state,
+            ace_handler_cls=AceStepHandler,
+            llm_handler_cls=LLMHandler,
+        )
+        cache_root = runtime.cache_root
```

Ruff (0.15.2)

[error] Local variable `cache_root` is assigned to but never used

Remove assignment to unused variable `cache_root`

(F841)